src/AppBridgeBlock.spec.ts

```diff
@@ -339,7 +339,7 @@ describe('AppBridgeBlockTest', () => {
     });
 
     test('updateColorPalette with default language', () => {
-        const frontifyColorPalette = FrontifyColorPaletteDummy.with(COLOR_PALETTE_ID);
+        const frontifyColorPalette = ColorPaletteDummy.with(COLOR_PALETTE_ID);
```

Comment:
I think we should update the constant as well here: 
```suggestion
        const colorPalette = ColorPaletteDummy.with(COLOR_PALETTE_ID);
```